### DISCUSSION TITLE
Welcome to PDS4-LDD-Issue-Repo Discussions!

### DISCUSSION THREAD

**Discussion by jordanpadams** [pds-data-dictionaries/ldd-management#186] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wkiri** [comment]
Hi everyone!  I'm at JPL in the Machine Learning and Instrument Autonomy group.  I also serve as the PDS Imaging Node Technologist and employ machine learning to enable content-based search in the Image Atlas.  I'm managing the Machine Learning Analysis LDD for PDS4.  Great to meet you all!

**Comment by stevenlujpl** [comment]
My name is Steven Lu. I am a member of the Machine Learning and Instrument Autonomy group at JPL, and I am supporting the development of the Machine Learning PDS4 LDD.
